docs/sources/developers/plugins/sign-a-plugin.md

```diff
@@ -32,7 +30,16 @@ To verify ownership of your plugin, you need to generate an API key that you'll
 
 Public plugins need to be reviewed by the Grafana team before you can sign them.
 
-1. Submit your plugin for [review]({{< relref "publish-a-plugin/#publishing-your-plugin-for-the-first-time" >}})
+1. Submit your plugin for review. 
```

Comment:
We have a dedicated page with the topic on how to submit a plugin which contains more detailed information than here (with a screenshot). This reduced set of instructions might not replace all of that.

Reply:
@josmperez, you might want to track down the more detailed content and see if it makes sense to reuse the file at this location. However, there might be SEO consequences if we choose one over the other, so keep that in mind.

Reply:
Oddly, when this PR was made the link in question gave a 404, but now (as academo observes) it is working. I will deprecate this PR and create a new one soon for doc team review only which will contain the typo fix.